NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -82,6 +82,23 @@ static RichTextBoxTarget()
             DefaultRowColoringRules = rules.AsReadOnly();
         }
 
+        public static void ReInitializeAllTextboxes(Form form)
+        {
+            foreach (Target target in LogManager.Configuration.AllTargets)
+            {
+                RichTextBoxTarget rtbt = target as RichTextBoxTarget;
+                if (rtbt != null && rtbt.FormName == form.Name && rtbt.TargetRichTextBox == null)
```

Comment:
please use English names for variables, like "textbox" etc (I know it was not before the 2.1.0).